### DISCUSSION TITLE
Uso da APyB pela PythOnRio

### DISCUSSION THREAD

**Discussion by biancarosa** [apyb/comunidade#29] [Geral]
Pessoal,

Estamos voltando com os encontros e queria organizar um pouco como a comunidade de Python carioca tem que se organizar. 
Uma das coisas que sempre foi dificuldade pra gente foi grana.
Eu estava pensando em como podemos operacionalizar doações pontuais e recorrentes, que podem ir diretamente pra conta da APyB, com algum tipo de acordo que se a gente precisar de algum suporte financeiro, podemos pedir reembolso pra APyB. Basicamente o que a gente não gastar no mês ficaria de doação pros outros eventos regionais, Python Sudeste ou PyBR. O que vocês acham? Não precisamos de uma conta separada pois é um evento pequeno.

**Comment by rougeth** [comment]
Oi @biancarosa, eu vou descrever como o processo hoje funciona para os eventos que acontecem anualmente:
1) Big Kahuna do evento preenche um formulário com seus dados e os dados do evento, e nesse formulário, criamos o código do evento (ex pybr2022). Esse código é usado para identificar todas as transações relacionadas ao evento.
2) Quando o evento precisa receber patrocínio ou realizar algum pagamento, temos uma automação que ajuda a manter controle, sempre usando o código do evento como id.
3) Após o fim do evento, todo dinheiro que restar do evento, fica como caixa para a APyB.

Nesse cenário, funcionaria se usássemos o mesmo processo para os encontros da PythOnRio? Criando um código novo para cada encontro, por exemplo `pythonrio202207` (julho). Nós nunca tivemos uma demanda de eventos mensais, então não sabemos se a APyB comportaria na forma atual, porém, podemos testar.

Além disso, você poderia compartilhar com a gente a entrada e saída dos últimos meses? Só para termos ideia de volume de transação (não precisa de detalhes, é só a quantidade mesmo).

**Comment by biancarosa** [comment]
Oi @rougeth! Hoje temos literalmente zero caixa, então a ideia é tentar iniciar esse processo pra gerar alguma grana tanto pra pythonrio quanto pra apyb. Acho que o fluxo pode funcionar, mas penso que talvez seja um pouco burocrático pra todos nós ter que resetar isso por evento. 

O que você acha de fazermos um teste pra `pythonrio2022` e fechamos anualmente o evento? Acho que isso encaixaria em um ótimo meio termo para nós 🤔

**Reply by rougeth** [reply]
Boa ideia @biancarosa, fica mais simples e já usa a infraestrutura existente. Acho que não precisamos mudar nada dessa forma, algum outro ponto @apyb/diretoria?

@biancarosa sobre os processos de emitir nota fiscal, pedir reembolso, etc - https://docs.google.com/document/d/1j0UwPbZrN1E8W3zdclzyIxaFPgztjLAATSdsPAizFX0/edit
